addons/partner_autocomplete/static/src/js/partner_autocomplete_core.js

```diff
@@ -328,5 +343,5 @@ export function usePartnerAutocomplete() {
             notification.add(title);
         }
     }
-    return { autocomplete, getCreateData, isVATNumber };
+    return { autocomplete, getCreateData, isVATNumber, isTAXNumber };
```

Comment:
There weren't any other occurrences of `isVATNumber`, so it doesn't need to be exported anymore.
```suggestion
    return { autocomplete, getCreateData, isTAXNumber };
```